packages/backend/discovery/espresso/ethereum/config.jsonc

```diff
@@ -18,7 +19,29 @@
         "stateHistoryFirstIndex",
         "getStateHistoryCount",
         "finalizedState"
-      ]
+      ],
+      "fields": {
+        "permissionedProver": {
+          "target": {
+            "permissions": [
+              {
+                "type": "configure",
+                "description": "relayer (prover) address that is allowed to call newFinalizedState() to prove the latest HotShot state to the Light Client contract on Ethereum."
+              }
+            ]
+          }
+        },
+        "owner": {
+          "target": {
+            "permissions": [
+              {
+                "type": "configure",
+                "description": "holds the power to change the contract state."
```

Comment:
This description is too gerenal. Check what functionality in contracts is e.g. "onlyOwner" and specify here what that owner can do.

Reply:
Here is updated version:
![image](https://github.com/user-attachments/assets/12701806-958e-416a-add3-e68cb7942839)

Reply:
https://github.com/l2beat/l2beat/pull/6007/commits/a12a9a480b1c6c4649afec9121db3f8c6fdc65dd